Jenkinsfile

```diff
@@ -9,14 +9,12 @@ pipeline {
         choice(name: "BUILD_TYPE", choices: ["Release", "Debug"], description: "")
```

Comment:
can we also update this list?

Reply:
curious, which configuration is now default for running tests on CI

Reply:
Jenkins takes the 1st as default unless otherwise selected. What should be in the list?

Reply:
I'd say we want here newly added `Component` and `Static`.

Also, we *must* run tests with enabled DCHECKs, which is not the case if `Release` is going to be picked by default. I suggest not to merge the PR unless we properly keep DCHECKs enabled for tests, otherwise it is a regression.

Reply:
Sounds good, can you please submit a review to the PR requesting changes? So It doesn't just appear as I've approved it.

Reply:
for now dcheck_always_on should be enabled with `--gn=dcheck_always_on:true` until we do a followup to create a PR specific static build config